app/dispatches/players/enlist.rb

```diff
@@ -15,13 +13,13 @@ def initialize(game:, player:)
 
     def assign_game_to_player!
       @player = Player.create!(nickname: nickname, game: game, life: Player::LIFE)
-      @game.update(state: 'running')
+      game.update(state: 'running')
```

Comment:
Fair enough. I'm not used to these ultra-thin models yet 😄. Leaving the whole enlisting process aside (in its own object), what you you think of encapsulating the `:state`attribute behind some dedicated methods such as `Game#running!`, `Game#awaiting_player!`, and so on?